__docs__/wonder-blocks-switch/switch.stories.tsx

```diff
@@ -82,46 +82,50 @@ export const Default: StoryComponentType = {
  * The `onChange` prop is optional in case the toggle will
  * be wrapped in a larger clickable component.
  */
-export const Controlled: StoryComponentType = () => {
-    const [checkedOne, setCheckedOne] = React.useState(false);
-    const [checkedTwo, setCheckedTwo] = React.useState(false);
+export const Controlled: StoryComponentType = {
+    render: function Render() {
+        const [checkedOne, setCheckedOne] = React.useState(false);
+        const [checkedTwo, setCheckedTwo] = React.useState(false);
 
-    return (
-        <View style={styles.column}>
-            <Switch checked={checkedOne} onChange={setCheckedOne} />
-
-            <Switch
-                testId="test-switch"
-                aria-label="test switch"
-                checked={checkedTwo}
-                onChange={setCheckedTwo}
-                icon={<PhosphorIcon icon={magnifyingGlassIcon} />}
-            />
-        </View>
-    );
-};
-
-Controlled.play = async ({canvasElement}) => {
```

Comment:
In previous versions of Storybook you had to manually navigate to each story and press "play" to run the tests, right?

Reply:
Right! and that made it hard to determine when we introduced a regression with these interaction tests. I also added the CI step to ensure that we don't introduce issues in the future.

NOTE: Storybook uses the[ test-runner](https://storybook.js.org/docs/writing-tests/test-runner) for this, but it will be replaced by this new test addon.

The previous runner used Jest + Playwright, and they wanted to switch to Vitest specially because they can run these tests in browsers as well as in headless mode. This is a new `vitest` feature that is also being refined in collaboration between the Storybook and vitest teams: https://vitest.dev/guide/browser/

Reply:
More context here: https://storybook.js.org/blog/storybook-test-sneak-peek/